src/util/debug.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+const debug = require('debug');
+const root = path.resolve(__dirname, '../');
+const pkg = require(path.resolve(root, '../package.json'));
+const toolName = pkg.name.split('/').pop();
+const tokenize = (...parts) => parts.join(':');
+
+const taggedLogger = tag => {
+    const logger = debug(tag);
+    logger.errorMsg = msg => `[${tag}] ${msg}`;
+    logger.sub = sub => taggedLogger(tokenize(tag, sub));
+    return logger;
+};
+module.exports = {
+    here(p) {
```

Comment:
Can we use a more descriptive name for this? `setDebugFile` or something. A single adverb doesn't really say much